config/webpack/build-aot.webpack.config.js

```diff
@@ -19,6 +19,14 @@ function getWebpackConfig(skyPagesConfig) {
   let commonConfig = common.getWebpackConfig(skyPagesConfig);
   commonConfig.entry = null;
 
+  // Since the preloader is executed against the file system during an AoT build,
+  // we need to remove it from the webpack config, otherwise it will get executed twice.
+  commonConfig.module.rules = commonConfig.module.rules
```

Comment:
I'd say using the boolean _does_ make it easier to read. You could make it even easier to read by defining the function into a well-named variable. Simply my two outside-looking-in cents.